test/index.js

```diff
@@ -129,18 +131,51 @@ test('micromark-extension-mdx-expression', (t) => {
               start: 3,
               end: 4,
               name: 'b',
-              loc: {start: {line: 1, column: 3}, end: {line: 1, column: 4}},
```

Comment:
These changes are due to the fact `acorn.Position` has `offset` property, it was ignored by `@ts-expect-error`.